pre_commit_hooks/no_commit_to_branch.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import print_function
+
+import argparse
+import sys
+
+from pre_commit_hooks.util import cmd_output
+
+
+def is_on_branch(protected):
+    branch = cmd_output('git', 'symbolic-ref', 'HEAD')
+    chunks = branch.strip().split('/')
+    return chunks[2] == protected
```

Comment:
oops sorry I didn't catch this at first, I think this should be:

```python
return '/'.join(chunks[2:]) == protected
```

branches can have slashes in them (maybe add a test for this too?)